java/util/src/main/java/com/google/protobuf/util/JsonFormat.java

```diff
@@ -96,42 +96,46 @@
       Logger.getLogger(JsonFormat.class.getName());
 
   private JsonFormat() {}
-  
+
   /**
-   * Creates a {@link Printer} with default configurations.
+   * Creates a {@link Printer} with default configurations. Default printing option is Pretty
```

Comment:
This comment is incomplete. The default configuration is a combination of different options. I suggest we just omit it here and move the detailed explanation to the comment of the omittingInsignificantWhitespace() method.